nitro-protocol/test/contracts/NitroAdjudicator/concludeAndTransferAllAssets.test.ts

```diff
@@ -44,8 +44,6 @@ const token = setupContract(
 ) as unknown as Token & Contract;
 
 const provider = getTestProvider();
-const chainId = process.env.CHAIN_NETWORK_ID;
```

Comment:
I think there is some file(s) specifying this global variable checked in? May want to remove an entry there, too.